app/services/trainees/create_from_csv_row.rb

```diff
@@ -92,6 +92,18 @@ def disabilities
       disabilities.map { |disability| ::Hesa::CodeSets::Disabilities::NAME_MAPPING[disability] }.compact
     end
 
+    def first_names
+      csv_row["First names"].presence || csv_row["First name"]
+    end
+
+    def middle_names
+      csv_row["Middle names"].presence || csv_row["Middle name"]
+    end
+
+    def last_name
+      csv_row["Last names"].presence || csv_row["Last name"]
+    end
+
```

Comment:
should this fix be in a separate commit?

Reply:
this was impacting the Goldsmiths import - we seem to get back variations of this column quite frequently.